server/repository/src/migrations/v1_07_01/stock_on_hand_add_item_name.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    let casting = if cfg!(feature = "postgres") {
+        "::BIGINT"
+    } else {
+        ""
+    };
+
+    sql!(
+        connection,
+        r#"
+        DROP VIEW stock_on_hand;
+
+        CREATE VIEW stock_on_hand AS
+        SELECT
+          'n/a' AS id,
+          items_and_stores.item_id AS item_id,
+          items_and_stores.item_name AS item_name,
```

Comment:
The requisition lines are generated by item_stats so had to modify the stock_on_hand view